javascript/src/Argument.ts

```diff
@@ -11,8 +11,8 @@ export default class Argument {
 
     if (argGroups.length !== parameterTypes.length) {
       throw new CucumberExpressionError(
-        `Group has ${argGroups.length} capture groups (${argGroups.map(
-          (g) => g.value
+        `Group has ${argGroups.length} capture groups (${argGroups.map((g) =>
+          g ? g.value : null
```

Comment:
Would `g?.value` work?

Reply:
I suppose we could say `g?.value || null`, but it's the same, really.